subsys/bluetooth/host/hci_core.c

```diff
@@ -3266,8 +3266,10 @@ static int le_init_iso(void)
 
 		net_buf_unref(rsp);
 	} else if (IS_ENABLED(CONFIG_BT_CONN)) {
-		LOG_WRN("Read Buffer Size V2 command is not supported."
-			"No ISO buffers will be available");
+		if (IS_ENABLED(CONFIG_BT_ISO_UNICAST) || IS_ENABLED(CONFIG_BT_ISO_BROADCASTER)) {
```

Comment:
should we need a helper kconfig for TX? I see these two are in a bunch of places.

Reply:
I agreed, and I thought we actually had that, but I couldn't find it. On the bright side, I just did: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/Kconfig#L121-L125

I'll post an update

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/68968